i18n/en.json

```diff
@@ -238,8 +238,8 @@
   "sysops-active": "This wiki has $1 active sysop (total $2).",
   "sysops-wait": "Maybe it's better to wait for the reaction of the local sysop.",
   "sysops-not-gs": "It is not a global sysop wiki.",
-  "sysops-more-than-3": "This wiki have more than 3 active sysop (total $1).",
-  "sysops-more-than-10": "This wiki have more than 10 active sysop.",
+  "sysops-more-than-3": "This wiki has more than 3 active sysop (total $1).",
```

Comment:
shouldn't it be `3 active sysops` too then?

Reply:
Of course. Good catch.